.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - 'master'
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    validation:
+        name: Validation
+        runs-on: ubuntu-20.04
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
```

Comment:
Any backport should respect the defined min/max as per composer
So ideally, 7.1 ... 8.1 if possible.

Reply:
@AsonUnique and me just discussed this a few minutes ago. You are way too fast! 😄

Reply:
No, a minor constraint upgrade is not a BC break
But of course only needs to be done if parts of the new language are used or older PHP versions are not testable anymore.